### ISSUE TITLE
Validation loss trend 

### ISSUE THREAD

**Issue by hlaahmed** [autonomousvision/transfuser#56]
Hii, I have a question regarding validation loss trend 
In my models validation loss is oscillating and there is no clear trend
So, I tried training and validating your late fusion goal based model, I didn’t change anything in your code and I trained on clear weather dataset but still I can’t detect a trend in validation loss 
While training loss is very nice 
Also what is the best validation loss you have had in late fusion goal based model
Mine is 0.33

**Comment by ap229997** [comment]
There was oscillation in our case as well but when we looked at the moving average we could see a downward trend. As for the validation loss value, I don't remember it.
